### PR TITLE
Added test cases for good parquet files

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/parquet/test/parquet-loader.spec.js [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
 import {validateLoader} from 'test/common/conformance';
 
 import {ParquetLoader, ParquetWorkerLoader} from '@loaders.gl/parquet';
 import {isBrowser, load, setLoaderOptions} from '@loaders.gl/core';
 
 import {SUPPORTED_FILES, UNSUPPORTED_FILES, ENCRYPTED_FILES, BAD_FILES} from './data/files';
+import {
+  ALL_TYPES_DICTIONARY_STUB,
+  ALL_TYPES_PLAIN_STUB,
+  ALL_TYPES_PLAIN_SNUPPY_STUB,
```
Comment (ibgreen) [inline modules/parquet/test/parquet-loader.spec.js:13]: snappy?
```suggestion
  ALL_TYPES_PLAIN_SNAPPY_STUB,
```

File: modules/parquet/test/parquet-loader.spec.js [hunk 2]
```diff
@@ -18,13 +34,71 @@ test('ParquetLoader#loader objects', (t) => {
...
 test('ParquetLoader#load', async (t) => {
   t.comment(`SUPPORTED FILES`);
+
   for (const {title, path} of SUPPORTED_FILES) {
     const url = `${PARQUET_DIR}/${path}`;
     const data = await load(url, ParquetLoader, {parquet: {url}, worker: false});
-    t.ok(data, `GOOD(${title})`);
+
+    switch (title) {
+      case 'alltypes_dictionary':
```
Comment (ibgreen) [inline modules/parquet/test/parquet-loader.spec.js:46]: This switch statement is starting to make this code rather complicated, getting hard to see what is going on.

I wonder if we should drop the `files.js` table and just load filenames directly here?

File: modules/parquet/test/stubs.js [hunk 3]
```diff
@@ -0,0 +1,568 @@
+/* eslint-disable camelcase */
+
+export const ALL_TYPES_DICTIONARY_STUB = [
```
Comment (ibgreen) [inline modules/parquet/test/stubs.js:3]: Not sure if calling these `_EXPECTED` instead of `_STUB` would be more clear?

**Review by ibgreen — APPROVED** [review]
Good stuff

File: modules/parquet/test/parquet-loader.spec.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/parquet/test/parquet-loader.spec.js]: Renamed

File: modules/parquet/test/stubs.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply modules/parquet/test/stubs.js]: Renamed.

**Pull request merged**